pkg/querier/lazy_querier.go

```diff
@@ -47,16 +42,6 @@ func (l lazyQuerier) Close() error {
 	return l.next.Close()
 }
 
-// Get implements ChunkStore for the chunk tar HTTP handler.
```

Comment:
Is this implemented elsewhere or the feature has gone away?

Reply:
I didn't find any usage of this functionality.